tasks/agent.py

```diff
@@ -72,6 +72,7 @@
     "telemetry",
     "orchestrator_pod",
     "orchestrator_ecs",
+    "cisco_sdwan"
```

Comment:
How does `AGENT_CORECHECKS` works, just to have a bit more context ?

Reply:
The list is used below to copy config examples / defaults
https://github.com/DataDog/datadog-agent/blob/b1f1eceb2a618684fb45965037ed098e7121bba4/tasks/agent.py#L320-L324